playbooks/host/ensure_host_with_randompassword.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Tests
```

Comment:
Playbook to ensure host with a random password

Reply:
Fixed